### PR TITLE
OSDOCS-7633: add storage migrator release note

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-7633

Link to docs preview:
https://65750--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-14-release-notes#microshift-4-14-storage

QE review:
- QE approved doc content [here](https://github.com/openshift/openshift-docs/pull/64470).

### CODE REVIEW

File: microshift_release_notes/microshift-4-14-release-notes.adoc [hunk 1]
```diff
@@ -84,10 +84,11 @@ With this release, updates for both minor releases and patch releases are suppor
 
 [id="microshift-4-14-storage"]
 === Storage
+With this release, the Migrator Controller embedded in {product-title} is used to update stored data to the latest storage version. You can update data without having to recreate custom resources (CRs). See xref:../microshift_storage/microshift-storage-migration.adoc#microshift-storage-migration[Storage migration using the Kube Storage Version Migrator] for details.
```
Comment (xenolinux) [inline microshift_release_notes/microshift-4-14-release-notes.adoc:87]:
```suggestion
With this release, the Migrator Controller embedded in {product-title} is used to update stored data to the latest storage version. You can update data without recreating custom resources (CRs). See xref:../microshift_storage/microshift-storage-migration.adoc#microshift-storage-migration[Storage migration using the Kube Storage Version Migrator] for details.
```

**Review by xenolinux — COMMENTED** [review]
I left one suggestion but otherwise looks good!

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

**Comment by abhatt-rh** [comment]
> I left one suggestion but otherwise looks good!
> 
> I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

@ShaunaDiaz @xenolinux : Peer review LGTM!

**Pull request merged**